pkg/gds/admin.go

```diff
@@ -222,7 +222,7 @@ func (s *Admin) setupRoutes() (err error) {
 			vasps.GET("/:vaspID", s.RetrieveVASP)
 			vasps.PATCH("/:vaspID", csrf, s.UpdateVASP)
 			vasps.DELETE("/:vaspID", csrf, s.DeleteVASP)
-			vasps.GET("/:vaspID/certificates", csrf, s.ListCertificates)
+			vasps.GET("/:vaspID/certificates", s.ListCertificates)
```

Comment:
I think this route does not require csrf protection, since it does not modify the state on the server.

Reply:
I agree, the RetrieveVASP GET on 222 already isn't using it so this is probably fine.